jgrapht-core/src/main/java/org/jgrapht/graph/builder/GraphBuilderBase.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * (C) Copyright 2015-2017, by Andrew Chen and Contributors.
+ *
+ * JGraphT : a free Java graph-theory library
+ *
+ * This program and the accompanying materials are dual-licensed under
+ * either
+ *
+ * (a) the terms of the GNU Lesser General Public License version 2.1
+ * as published by the Free Software Foundation, or (at your option) any
+ * later version.
+ *
+ * or (per the licensee's choosing)
+ *
+ * (b) the terms of the Eclipse Public License v1.0 as published by
+ * the Eclipse Foundation.
+ */
+package org.jgrapht.graph.builder;
+
+import org.jgrapht.*;
+
+/**
+ * A builder class for {@link Graph}.
+ * 
+ * @param <V> the graph vertex type
+ * @param <E> the graph edge type
+ * @param <G> type of the resulting graph
+ * 
+ * @author Andrew Chen
+ */
+public class GraphBuilderBase<V, E, G extends Graph<V, E>>
```

Comment:
I would recommend renaming this class to GraphBuilder. There are no classes left which extend this class.